source/plugin/wgen-index.rst

```diff
@@ -0,0 +1,70 @@
+================
+World Generation
+================
+
+World Generation in Minecraft is a complicated process, and Sponge adds its own complications too.
+Here we provide a basic overview of how it works for the interested reader.
+You may choose to skip this and move directly to the API details here:
+
+[Links to WGen API pages TBA]
+
+
+An Overview of World Generation in Minecraft
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+World generation in Minecraft is based on *chunks*. Whenever the game calls for a chunk which does not yet exist,
+a request is made to the world generator to provide that chunk. The world generation process to provide this 
+chunk is split into two distinct phases, named ``generation`` and ``population``.
+
+The Generation Phase
+--------------------
+
+The first phase in providing a chunk is the *Generation Phase*. This is primarily responsible for the 
+creation of the base terrain shape, the generation of biomes, and larger features such as villages and caves.
+
+.. Note::
+	Block placement in the generation phase takes place entirely within a buffer,
+	as the Chunk object is not created until the last step in the phase.
+
+In the beginning, there is nothing, and then the base generator populator creates the basic shape of the terrain.
+At this stage the world is entirely made of stone. Default generation is simply a heightmap generated from 
+perlin noise, then filled in - leaving air if higher than sea level, and water if below. This step can be overridden
+by calling ``setBaseGeneratorPopulator`` to provide your own custom populator to the world generator.
+
+Next, the biome generator populates a 2d biome buffer with the biomes that will correspond to each 1x1 column
```

Comment:
Just noticed these first two pieces are in the wrong order, The biomes are generated first as then the base terrain is then created based off of the information from the biomes. So the order should follow:

1. The BiomeGenerator populates a 2d buffer
2. The base GenerationPopulator creates the base terrain shape
3. the top layers of terrain are replaced according to the GroundCoverLayers
4. Etc.